src/main/java/thunder/hack/injection/MixinHandledScreen.java

```diff
@@ -281,8 +281,12 @@ private void mouseClicked(double mouseX, double mouseY, int button, CallbackInfo
         }
         for (Render2DEngine.Rectangle rect : clickableRects.keySet()) {
             if (rect.contains(mouseX, mouseY)) {
-                mc.interactionManager.clickSlot(mc.player.currentScreenHandler.syncId, clickableRects.get(rect), 0, SlotActionType.PICKUP, mc.player);
+                if (ModuleManager.tooltips.shulkerRegear) {
```

Comment:
Maybe ```shulkerRegearShiftMode``` instead of  ```shulkerRegear``` ?

Reply:
yeah that's a goof lol, i just fixed it on my fork. was tired writing it. Should I open a new pr?
sorry i'm not on gh much

Reply:
Yea